python-challenges/email-scheduler/devashmit/scheduler.py

```
@@ -0,0 +1,106 @@
+        # Demo mode: send to a sample list
+        recipients = [
+            {"email": "alice@example.com", "name": "Alice"},
+            {"email": "bob@example.com",   "name": "Bob"},
+        ]
+        print("  (recipients.csv not found — using demo list)\n")
+
+    print(f"  Sending to {len(recipients)} recipient(s)...\n")
+
+    for r in recipients:
+        email   = r.get("email", "").strip()
+        if not email: continue
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)